src/styles/all-styles/card.scss

```diff
@@ -0,0 +1,114 @@
+.catalog {
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
fix it 
![image](https://user-images.githubusercontent.com/97547824/217911785-e2178bcc-f5ee-42c9-9c39-7d90c1d9f0fe.png)